### PR TITLE
Adding event listeners @blur and @keyup.enter to schema

### PR DESCRIPTION
**Pull request by paulogiacomelli**

### CODE REVIEW

File: src/fields/core/fieldInput.vue [hunk 1]
```diff
@@ -38,168 +39,167 @@
...
-				case "number":
-				case "range":
-					if (isNumber(parseFloat($event.target.value))) {
-						value = parseFloat($event.target.value);
-					}
-					break;
-			}
-			this.value = value;
-		},
-		onBlur() {
```
Comment (dtorres-sf) [inline src/fields/core/fieldInput.vue]: It looks like you are removing functionality here. I think we should keep this `onBlur` and in this `onBlur` call `this.onDataAccept`

**Review by dtorres-sf — CHANGES_REQUESTED** [review]
Just one change for the input `onBlur`. In general, when working with third party libraries like this, you should stick to their default linting and style guidelines. Basically not make all the formatting/whitespace changes. The reason is because they will continue to stick to that style and it can over complicate merges if we ever want to pull back in their changes. If it is not easy to undo, don't worry about it this time, but something to be mindful of going forward.

File: src/fields/core/fieldInput.vue (hunk 1 above)
Comment (paulogiacomelli) [reply src/fields/core/fieldInput.vue]: Thanks for the heads up. I cleaned up! I just called the this.onDataAccept after the current functionality.

File: src/fields/core/fieldInput.vue [hunk 2]
```diff
@@ -6,6 +6,7 @@
 		:value="value",
 		@input="onInput",
 		@blur="onBlur",
+		@keyup.enter="onDataAccept",
```
Comment (dtorres-sf) [inline src/fields/core/fieldInput.vue]: This should go through the same path as `blur` which is just flushing out any debouncing that is being done.

```suggestion
		@keyup.enter="onBlur",
```

**Review by dtorres-sf — APPROVED** [review]

**Pull request merged**